ibis/backends/mysql/__init__.py

```diff
@@ -230,13 +230,15 @@ def _get_temp_view_definition(self, name: str, definition: str) -> str:
             expression=definition,
         )
 
+    # TODO(kszucs): should be moved to the base sqlglot backend
```

Comment:
Not really, most backends don't support it.

Reply:
Removed the note.